src/components/FindProjectTutorial.vue

```diff
@@ -0,0 +1,321 @@
+        )
+      }
+    },
+    fetchTutorialProject(groupId: string | undefined) {
+      if (this.tutorial?.projectId && groupId) {
+        onValue(
+          getTasksRef(this.tutorial.projectId, groupId),
+          (snapshot) => {
+            const data = snapshot.val()
+            this.tasks = decompressTasks(data)
+            this.results = this.tasks.reduce(
+              (acc, val) => {
+                // Fill all the result with initial value
+                acc[val.taskId] = this.options[0].value
+
+                return acc
+              },
+              {} as Record<string, number>,
+            )
+          },
+          (error) => {
+            console.error('Error fetching tasks for the tutorial', error)
+          },
+          { onlyOnce: true },
+        )
+      }
+    },
+    nextTask() {
+      if (!this.isLastTask) {
+        if (this.isLastTask) {
+          return
+        }
+
+        this.selectedTasks = {}
+        this.currentTaskIndex += 1
+        this.userAttempts = 0
+        this.answersRevealed = false
+      }
+    },
+    updateResult(taskId: string) {
+      if (!this.answersRevealed) {
+        this.userAttempts += 1
+        const currentResult = this.results[taskId]
+
+        if (!isDefined(currentResult)) {
+          this.results[taskId] = this.options[0].value
+          return
+        }
+
+        const newValue = this.optionMap[this.optionMap[currentResult].nextOptionKey].value
+
+        this.results[taskId] = newValue
+      }
+    },
+    handleTileClicked(taskId: string) {
+      this.updateResult(taskId)
+    },
+    handleTileSelected(newValue: boolean, taskId: string) {
+      this.selectedTasks[taskId] = newValue
+    },
+    showAnswer() {
+      this.answersRevealed = true
+      this.taskList.forEach((task) => {
+        this.results[task.taskId] = task.referenceAnswer
+      })
+    },
+  },
+  emits: ['tutorialComplete'],
+  mounted() {
+    goOnline(db)
+    this.fetchTutorialGroups()
+    // this.fetchTutorialProject();
+  },
+})
+</script>
+
+<template>
+  <v-container>
+    <v-row>
+      <v-col>
+        <div>
+          {{ instructionMessage }}
+        </div>
+      </v-col>
+    </v-row>
+    <v-row v-if="alertContent">
+      <v-col>
+        <v-alert
+          density="compact"
+          border="start"
+          variant="tonal"
+          :type="alertContent.type"
+          :title="alertContent.title"
+          :text="alertContent.text"
+          :icon="alertContent.icon"
+        />
+      </v-col>
+    </v-row>
+    <v-row justify="center">
+      <v-col md="6">
+        <div class="image-grid">
+          <find-project-task
+            v-for="task in taskList"
+            :key="task.taskId"
+            :task="task"
+            :optionMap="optionMap"
+            :selected="selectedTasks[task.taskId]"
+            :value="results[task.taskId]"
+            @updateValue="handleTileClicked"
+            @updateSelected="handleTileSelected"
+          />
```

Comment:
FindProjectTask is called without defining the tileSize prop here. As a result, there is often the need to scroll within the tutorial tab on smaller screens.

In FindProject, tileSize is evaluated based on available window space -- I am wondering if we can find a way to dynamically determine a reasonable tileSize for tiles in the tutorial as well.

Reply:
Not sure how this would work for tutorials, as the scenes are defined in the tutorials as group of 6 tiles

Reply:
We've used viewport height based size for the tiles in tutorial. So, it's relatively better now! @ofr1tz let us know if this is sufficient for the moment.

Reply:
This is a great improvement, looks good to me!